### DISCUSSION TITLE
[Question] streaming html

### DISCUSSION THREAD

**Discussion by mtsewrs** [uNetworking/uWebSockets.js#209] [Q&A] [answered]
When streaming html is fine to just use `res.write` instead of `res.tryEnd`? Wondering for performance reasons and what not. The stream comes from ReactJS `renderToNodeStream` if that helps.

**Comment by ghost** [comment, marked as answer]
Yes if you write chunks of reasonable size. But write is worse than tryEnd. You should test what works for you

**Comment by mtsewrs** [comment]
@alexhultman ok, I guess I have to use `write` either way since I need to write stuff after the stream is done and `tryEnd` will end the response
